src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -172,17 +181,12 @@ class MultipleFields extends Component {
       props: { label, value: values, component, onChange },
     } = this;
 
-    // values && values.length is to validate the object is not null and not an empty array, respectively
-    // the last `&& values` is to make sure after validation, it always return `values`  instead of `values.length`
-    const usedValues = (values && values.length && values) || [
```

Comment:
Removed the local variable usedValues and using only values. With this, the actual changes in the widget would be directly reflected in the redux state thus keeping a single source of truth as values